src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.xpro.QA.yaml

```diff
@@ -4,18 +4,14 @@ encryptedkey: AQICAHijXuVxVlAL6bY9xCOrzO3YYhFlQBPt6jNyJGkhYu+q4QHxoDDVZ4kHnQtnul
 config:
   aws:region: us-east-1
   consul:address: https://consul-xpro-qa.odl.mit.edu
-  edxapp:additional_waffle_flags:
-  - ["course_experience.relative_dates", "--create", "--superusers", "--everyone"]
+  edxapp:waffle_flags:
+  - ["course_experience.relative_dates", "--create", "--superusers", "--everyone",
+    "--staff", "--authenticated"]
   - ["courseware.courseware_mfe", "--create", "--superusers", "--everyone"]
-  - ["discussions.pages_and_resources_mfe", "--create", "--superusers", "--everyone"]
```

Comment:
Why is `discussions.pages_and_resources_mfe` being removed?

Reply:
So this is purely a snapshot of the flag states from when I was working on this PR. It probably shouldn't be removed.

Reply:
Yeah, we'll want to make sure that xPro QA matches what is being set in CI. Once that's done this should be good to merge.